docs/docs/playground/isConnected.mdx

```diff
@@ -0,0 +1,32 @@
+---
+id: isConnected
+title: isConnected
+---
+
+#### `isConnected()`
+
+<strong>Test Lyra's `isConnected` method:</strong>
+
+import { useState } from "react";
```

Comment:
I think these need ``` codeblock. I am getting `Can't resolve '@stellar/lyra-api' in '/Users/jeesunkim/Code/lyra/docs/docs/playground'`

Reply:
ah good catch - you actually need to build `@stellar/lyra-api` locally as these playground pages run the `@stellar/lyra-api` library. Added this to README:
```
First build `@stellar/lyra-api` (a dependency of this project) by cd'ing to `@stellar/lyra-api` and running `yarn build`.

Then, come back to this directory and run:

```
$ yarn start
```
```